src/app/(screens)/clEvent.tsx

```diff
@@ -94,10 +95,25 @@ export default function ClEventDetail(): JSX.Element {
                       ]),
                 ...(clEvent?.location != null && clEvent?.location !== ''
                     ? [
-                          {
-                              title: t('pages.event.location'),
-                              value: clEvent?.location,
-                          },
+                          Object.assign(
+                              {
+                                  title: t('pages.event.location'),
+                                  value: clEvent?.location,
+                              },
+                              isValidRoom(clEvent.location)
+                                  ? {
+                                        onPress: () => {
+                                            router.navigate({
```

Comment:
Please use `dismissTo` instead of navigate, which enables better performance and animation on iOS.
```suggestion
                                            router.dismissTo({
```